service/task_history.go

```diff
@@ -516,10 +516,9 @@ func getTaskDrawerItems(displayName string, variant string, reverseOrder bool, v
 		return nil, errors.Wrap(err, "error getting sibling tasks")
 	}
 
-	for i := range tasks {
-		if err := tasks[i].MergeNewTestResults(); err != nil {
-			return nil, errors.Wrap(err, "error merging test results")
-		}
+	tasks, err = task.MergeTestResultsBulk(tasks)
```

Comment:
Since the task history drawer only displays tests with errors, can we query for just those tests?

Reply:
Fixed. I did it in a roundabout ish way since I want to use this for other queries too